#163530737 setup database

haywhyze opened the pull request:
#### What does this PR do?
This PR sets up the database
#### Description of Task to be completed?
- install necessary modules
- add scripts to the models to dynamically create all tables.
- add scripts to the models to seed initial data
#### How should this be manually tested?
- clone this repository by executing `git clone https://github.com/haywhyze/politico.git`
- cd to the cloned local repository `cd politico`
- run `git checkout ch-setup-database-16350737`
- install all dependencies by running `npm install`
- start the server with `npm start`
- when the server is up, you can test using Postman or any other related service.

#### What are the relevant pivotal tracker stories?
[#16350737](https://www.pivotaltracker.com/story/show/16350737)

haywhyze committed:
feat(server): implement get one party

- write test scripts to test the getOne party functionalilty
- create a middleware to validate party ID
- create getOne method to handle get request to `/parties/<party-id>`
- add the getOne method and the validateID middleware to the router

[delivers #163423238]

haywhyze committed:
chore(docs): Update readme

- update readme to show test coverage and code maintanabilty report from CI tools

[finishes #163504828]

haywhyze committed:
chore(server): setup database

- install necessary modules
- add scripts to the models to dynamically create all tables.
- add scripts to the models to seed initial data

[finishes #163530737]

haywhyze committed:
chore(server): setup database

- removed inline eslint errors

[finishes #163530737]

akinwale-habibullah reviewed (commented):
Please add installation explanation in your readme, so I know what variables I must configure to get started with your repo. @haywhyze

haywhyze committed:
chore(tools): setup database

- update readme and .env.sample to include more information on integrating a database.

[finishes: 163530737]

akinwale-habibullah commented on server/models/db.js:
```diff
@@ -0,0 +1,15 @@
+  connectionString,
+});
+
+module.exports = {
```
Use es6 export construct

akinwale-habibullah reviewed (commented):
Kindly reflect on the feedback during our discussion and implement same.

haywhyze committed:
chore(tools): setup database

- use export default on db config.
- set passwords in .env
- make a loop of seeded data
- break routers into modules
- put a prompt for success on db-init

[finishes #163530737]
